chopper_generator/pubspec.yaml

```diff
@@ -24,6 +24,6 @@ dev_dependencies:
   pedantic: ^1.9.2
 
 # Uncomment for development
-# dependency_overrides:
-#   chopper:
-#     path: ../chopper
+dependency_overrides:
+  chopper:
+    path: ../chopper
```

Comment:
Comment again :D

Reply:
oops! I thought I had pushed the update with this reverted.